code/game/objects/items/weapons/extinguisher.dm

```diff
@@ -42,7 +42,8 @@
 /obj/item/weapon/extinguisher/New()
 	..()
 	create_reagents(max_water)
-	reagents.add_reagent("water", max_water)
+	reagents.add_reagent("water", (max_water - 10))

```

Comment:
Clever way to handle smaller extinguishers, maybe we should make it a 1:4 ratio?